biz.aQute.bndlib/src/aQute/bnd/osgi/resource/ResourceImpl.java

```diff
@@ -51,7 +51,8 @@ void setCapabilities(List<Capability> capabilities) {
 
 	@Override
 	public List<Capability> getCapabilities(String namespace) {
```

Comment:
We seem to basically reimplement MultiMap? Shouldn't we use that one and do any optimizations there?

Reply:
Maybe a separate PR? But best not to conflate with this PR.